server/api/team/team.spec.js

```diff
@@ -62,3 +68,84 @@ describe('GET /api/teams/:id/players', function() {
       });
   });
 });
+
+
+describe ('POST /api/teams/fill', function() {
+  var account1 = {
+    email: 'test1@test.com',
+    password: 'test',
+    role: 'admin'
+  };
+
+  var account2 = {
+    email: 'test2@test.com',
+    password: 'testing',
+    role: 'commish'
+  };
+
+  before(function(done) {
+    // Clear db before testing
+    db.Sport.destroy({}, {truncate: true}).then(function() {
+      return db.League.destroy({}, {truncate: true})
+    }).then(function() {
+      return db.User.destroy({}, {truncate: true})
+    }).then(function() {
+      return db.Team.destroy({}, {truncate: true})
+    }).then(function() {
+      return db.Player.destroy({}, {truncate: true})
+    }).then(function() {
+      return db.PlayerAssignment.destroy({}, {truncate: true})
+    }).then(function() {
+      return done();
```

Comment:
I don't think you need to return done() here -- I think just calling done() is enough for the test framework.